src/matlab2tikz.m

```diff
@@ -4500,6 +4500,34 @@ function warnFacetedInterp(m2t)
     end
 end
 % ==============================================================================
+function   cbarStyleOptions = getColotBarTicks(handle, cbarStyleOptions)
+if strcmp( get(handle,'TickLabelsMode') , 'manual')
+
+    %ColorBar TickLabels 
+    TickLabels= get(handle,'TickLabels');
+    TickLabels_option ='{';
+    for e=1:length(TickLabels)
+        TickLabels_option = [ TickLabels_option ' {' TickLabels{e} '},'];
+    end
+    TickLabels_option(end)='}';
+
+    cbarStyleOptions=cat(1,cbarStyleOptions, {'yticklabels',TickLabels_option});
```

Comment:
This seems really cumbersome and I would guess it fails for empty ticks?

Please have a look at
```matlab
function [options] = getAxisTicks(m2t, handle, axis, options)
```

Basically that is exactly what you would want here.